src/views/admin/PendingCredits.vue

```diff
@@ -57,12 +58,20 @@
 
   @Options({
     name:"PendingCredits",
+    data() {
+      return {
+        hasLoadingError: false,
+      }
+    },
     mounted() {
       this.updatePendingCreditRequests()
     },
     computed: {
       pendingCreditRequests(): Array<any> {
         return this.$store.state.lokapi.pendingCreditRequests
+      },
+      hasLoadingError(): Array<any> {
+        return this.$store.state.lokapi.pendingCreditRequestsError
```

Comment:
useless call on a non-existing variable in the store

Reply:
Thanks for spotting the error.
Changed.